src/optimistic-oracle-v2/implementation/ManagedOptimisticOracleV2.sol

```diff
@@ -104,6 +106,7 @@ contract ManagedOptimisticOracleV2 is
     mapping(bytes32 => DisableableAddressWhitelistInterface) public customProposerWhitelists;
 
     // Admin controlled bounds limiting the changes that can be made by request managers.
+    mapping(IERC20 => uint256) public minimumBonds; // Minimum bonds for a given currency.
```

Comment:
Though gas costs are not priority here, it might be better to use a single base slot for a min/max slot. This avoids the second keccak when deriving the raw slot, especially as we read both min and max when validating.
The downside is we need another struct, similar  to `BondCapInfo` but without the IERC20 prop.